index.html

```diff
@@ -107,10 +107,10 @@ <h2> Read our most recent blogs </h2>
             <div class="card">
               <img src="./assets/cumbria.jpg" class="card-img-top" alt="Python">
               <div class="card-body">
-                <h5 class="card-title">Blog 1</h5>
-                <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+                <h5 class="card-title">Into the Wild</h5>
+                <p class="card-text">Making the most of the sunny weather, Tiegen and her partner explore the Lake District. Camping, hiking and swimming- all the good stuff!</p>
```

Comment:
Corrected my name, there's a few more across the page to correct too.
```suggestion
                <p class="card-text">Making the most of the sunny weather, Tiegan and her partner explore the Lake District. Camping, hiking and swimming- all the good stuff!</p>
```

Reply:
I made those changes for you as it is a quick one to change with a commit!